x/erc20/keeper/migrations.go

```diff
@@ -11,3 +18,8 @@ func NewMigrator(keeper Keeper) Migrator {
 		keeper: keeper,
 	}
 }
+
+// Migrate1to2 migrates from consensus version 1 to 2.
+func (m Migrator) Migrate1to2(ctx sdk.Context) error {
+	return nil
+}
```

Comment:
seams that this is not needed, there is no state machine change / migration in the erc20 module.